File: java8/1041.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		Scanner leitor = new Scanner(System.in);
		double [] coordenadas = new double [2];
		String [] coodenadasString = new String [2];
		
		String entradasStr = leitor.nextLine();
		coodenadasString = entradasStr.split(" ");
		for (int k = 0; k < 2; k++) {
			coordenadas [k] = Double.parseDouble(coodenadasString[k]);
		}
				
		if (coordenadas[0] == 0 && coordenadas[1] == 0) {
			System.out.println("Origem");
		} else if (coordenadas [0] == 0) {
			System.out.println("Eixo Y");
		} else if (coordenadas[1] == 0){
			System.out.println("Eixo X");
		} else { 
			if(coordenadas[0] > 0 && coordenadas[1] > 0) {
				System.out.println("Q1");
			} else if(coordenadas[0] < 0 && coordenadas[1] > 0) {
				System.out.println("Q2");
			} else if (coordenadas[0] < 0 && coordenadas[1] < 0) {
				System.out.println("Q3");	
			} else if (coordenadas[0] > 0 && coordenadas[1] < 0) {
				System.out.println("Q4");
			}
		}
		
		leitor.close();
	}
}

File: python3/1041.py
# -*- coding: utf-8 -*-
entrada = input().split(" ")
x = float(entrada[0])
y = float (entrada[1])

if (x == 0 and y == 0):
    print("Origem")
elif (x == 0):
    print("Eixo Y")
elif (y == 0):
    print("Eixo X")
else:
    if(x > 0 and y > 0):
        print("Q1")
    elif(x < 0 and y > 0):
        print("Q2")
    elif (x < 0 and y < 0):
        print("Q3")
    elif (x > 0 and y < 0):
        print("Q4")